src/Helpers/LeanCode.UserIdExtractors/Extractors/GuidUserIdExtractor.cs

```diff
@@ -0,0 +1,14 @@
+using System.Security.Claims;
+
+namespace LeanCode.UserIdExtractors.Extractors;
+
+internal class GuidUserIdExtractor : StringUserIdExtractor, IUserIdExtractor<Guid>
```

Comment:
`public sealed` IMO (and others). We gain nothing from hiding it.

Reply:
Plus, I don't think we should inherit the `StirngUserIdExtractor`. There isn't really any connection between the two.